src/components/Books.js

```diff
@@ -43,46 +44,47 @@ const Books = () => {
   };
 
   return (
-    <>
+    <div className="container">
       <ul className="books-list">
         {books.map((book) => (
-          <li key={book.item_id} className="py-3 d-flex justify-content-between">
-            <div className="d-flex flex-column">
-              <span>{book.category}</span>
-              <span>{book.title}</span>
-              <span>Anonymous</span>
-              <div>
-                <button type="button">Comment</button>
-                <button type="button" id={book.item_id} onClick={removeFromStore}>Remove</button>
-                <button type="button">Edit</button>
+          <li key={book.item_id} className="row book-item ps-2">
+            <div className="d-flex flex-column col-md-5 mt-4">
+              <span className="book-category mb-2">{book.category}</span>
+              <span className="book-title">{book.title}</span>
+              <span className="book-author">Anonymous</span>
+              <div className='mt-3'>
+                <button type="button" className="book-btn me-3">Comment</button>
+                <button type="button" className="book-btn me-3" id={book.item_id} onClick={removeFromStore}>Remove</button>
+                <button type="button" className="book-btn me-3">Edit</button>
               </div>
             </div>
-            <div className="d-flex flex-column">
-              <span>
-                64%
-              </span>
-              <span>Completed</span>
+            <div className="d-flex col-md-3">
```

Comment:
- [ ] Your project looks great, but to meet project requirements, vertical line between `book progress chart` and `book's detail card`.

Please find this vertical line from Zeplin snapshot below
<img width="303" alt="Screen Shot 2021-11-06 at 21 11 50" src="https://user-images.githubusercontent.com/60043698/140616283-2d184097-1563-4a4a-8c30-45e3efb25607.png">